tensorflow_datasets/image/cifar10_1.py

```diff
@@ -0,0 +1,148 @@
+
+"""Cifar-10.1 dataset"""
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+from tensorflow_datasets.core import api_utils
+import tensorflow_datasets.public_api as tfds
+
+import numpy as np
+
+_CITATION = """\
+@article{recht2018cifar10.1,
+  author = {Benjamin Recht and Rebecca Roelofs and Ludwig Schmidt and Vaishaal Shankar},
+  title = {Do CIFAR-10 Classifiers Generalize to CIFAR-10?},
+  year = {2018},
+  note = {\\url{https://arxiv.org/abs/1806.00451}},
+}
+
+@article{torralba2008tinyimages, 
+  author = {Antonio Torralba and Rob Fergus and William T. Freeman}, 
+  journal = {IEEE Transactions on Pattern Analysis and Machine Intelligence}, 
+  title = {80 Million Tiny Images: A Large Data Set for Nonparametric Object and Scene Recognition}, 
+  year = {2008}, 
+  volume = {30}, 
+  number = {11}, 
+  pages = {1958-1970}
+}
+"""
+
+_DESCRIPTION = """\
+The CIFAR-10.1 dataset is a new test set for CIFAR-10. CIFAR-10.1 contains roughly 2,000 new test images 
+that were sampled after multiple years of research on the original CIFAR-10 dataset. The data collection 
+for CIFAR-10.1 was designed to minimize distribution shift relative to the original dataset. We describe 
+the creation of CIFAR-10.1 in the paper "Do CIFAR-10 Classifiers Generalize to CIFAR-10?". 
+The images in CIFAR-10.1 are a subset of the TinyImages dataset. 
+There are currently two versions of the CIFAR-10.1 dataset: v4 and v6.
+"""
+
+_DL_URL = "https://github.com/modestyachts/CIFAR-10.1/blob/master/datasets/cifar10.1_{}_data.npy?raw=true"
+
+_DATA_OPTIONS = ["v4", "v6"]
+
+
+class Cifar10_1Config(tfds.core.BuilderConfig):
+  """BuilderConfig for Cifar-10.1"""
+
+  @api_utils.disallow_positional_args
+  def __init__(self, data=None, **kwargs):
+    """Constructs a Cifar10_1Config.
+    Args:
+      data: `str`, one of `_DATA_OPTIONS`.
+      **kwargs: keyword arguments forwarded to super.
+    """
+    if data not in _DATA_OPTIONS:
+      raise ValueError("data must be one of %s" % _DATA_OPTIONS)
+
+    super(Cifar10_1Config, self).__init__(**kwargs)
+    self.data = data
+
+
+class Cifar10_1(tfds.core.GeneratorBasedBuilder):
+  """Cifar-10.1 dataset"""
+
+  BUILDER_CONFIGS = [
+      Cifar10_1Config(
+          name="v4",
+          description=(
+              "It is the first version of our dataset on which we tested any classifier. As mentioned above, this "
+              "makes the v4 dataset independent of the classifiers we evaluate. The numbers reported in the main "
+              "sections of our paper use this version of the dataset. It was built from the top 25 TinyImages "
+              "keywords for each class, which led to a slight class imbalance. The largest difference is that ships "
+              "make up only 8% of the test set instead of 10%. v4 contains 2,021 images."),
+          version=tfds.core.Version(
+              "1.0.0", experiments={tfds.core.Experiment.S3: False}),
```

Comment:
Remove `, experiments={tfds.core.Experiment.S3: False}`.
New datasets should use S3.